### DISCUSSION TITLE
On a Windows PC keyboard, what function does the shortcut "Control+C" perform?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#182] [Galaxy Trivia] [answered]
**Hint:** ©

**Comment by thenithinbalaji** [comment, marked as answer]
Control+C is a common computer command. It is generated by pressing the C key while holding down the Ctrl key on most computer keyboards. In graphical user interface environments that use the control key to control the active program, control+C is often used to copy the highlighted text to the clipboard.

**Reply by ParanoidUser** [reply]
Salut! 😋 Congrats on your well deserved [x3 Galaxy Brain](https://github.com/thenithinbalaji?tab=achievements&achievement=galaxy-brain) badge! 16 more answers for the gold [x4]. Keep it up!

**Comment by thuongtruong1009** [comment]
On a Windows PC keyboard, the shortcut "Control+C" performs the function of copying selected text or other content to the clipboard. This keyboard shortcut is often used in conjunction with the "Control+V" shortcut, which pastes the copied content from the clipboard into another location.
